tests/resources/VCH-Util.robot

```diff
@@ -725,3 +725,12 @@ Assert
     [Arguments]  ${status}  ${msg}
     ${envExists}=  Run Keyword And Return Status  Environment Variable Should Be Set  FAST_FAILURE
     Run Keyword If  ${envExists}  Run Keyword If  %{FAST_FAILURE}  Run Keyword Unless  ${status}  Fatal Error  ${msg}
+
+Manually Cleanup VCH
+    [Arguments]  ${vch-name}
+    ${out}=  Run Keyword And Ignore Error  Run  govc vm.destroy ${vch-name}
+    ${out}=  Run Keyword And Ignore Error  Run  govc pool.destroy host/*/Resources/${vch-name}
+    ${out}=  Run Keyword And Ignore Error  Run  govc datastore.rm ${vch-name}
+    ${out}=  Run Keyword And Ignore Error  Run  govc host.portgroup.remove ${vch-name}-bridge
+    ${out}=  Run Keyword And Ignore Error  Run  govc ls | grep vm
+    ${out}=  Run Keyword And Ignore Error  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run  govc object.destroy  ${out}/${vch-name}
```

Comment:
This whole keyword needs to use vic-machine delete --name NOT govc.

Reply:
this govc pattern of cleanup is only used as a last resort for dangling cleanups.  we generally want to defer to vic-machine delete whenever possible